insalan/user/models.py

```diff
@@ -146,13 +146,15 @@ def send_email_confirmation(user_object: User):
         # TODO Give a frontend page instead of direct API link
         send_mail(
             _("Confirmez votre courriel"),
-            _("Confirmez votre adresse de courriel en cliquant sur ")
-            + "http://api."
-            + getenv("WEBSITE_HOST", "localhost")
-            + reverse("confirm-email", kwargs={"user": user, "token": token}),
+            "",
```

Comment:
This whole thing is still breaking tests AFAIK, and you can remove it to let #94 do the work of changing the URL